### ISSUE TITLE
Use of deprecated methods

### ISSUE THREAD

**Issue by Robbepop** [paritytech/cargo-contract#84]
When compiling current `cargo-contract` master I receive the following warnings:

```
   Compiling cargo-contract v0.7.0 (/home/robrob/.cargo/git/checkouts/cargo-contract-0e593efc9d3a70d5/169c65b)
warning: use of deprecated associated function `zip::ZipWriter::<W>::start_file_from_path`: by stripping `..`s from the path, the meaning of paths can change. Use `start_file` instead.
  --> build.rs:96:17
   |
96 |             zip.start_file_from_path(name.as_path(), options)?;
   |                 ^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

warning: use of deprecated associated function `zip::ZipWriter::<W>::add_directory_from_path`: by stripping `..`s from the path, the meaning of paths can change. Use `add_directory` instead.
   --> build.rs:103:17
    |
103 |             zip.add_directory_from_path(name.as_path(), options)?;
    |                 ^^^^^^^^^^^^^^^^^^^^^^^

warning: 2 warnings emitted
```

As well as the following errors when compiling `contract-metadata`:

```
   Compiling contract-metadata v0.1.0 (/home/robrob/.cargo/git/checkouts/cargo-contract-0e593efc9d3a70d5/169c65b/metadata)
warning: use of deprecated associated function `zip::read::ZipFile::<'a>::sanitized_name`: by stripping `..`s from the path, the meaning of paths can change.
                You must use a sanitization strategy that's appropriate for your input
  --> src/cmd/new.rs:60:41
   |
60 |         let outpath = out_dir.join(file.sanitized_name());
   |                                         ^^^^^^^^^^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

warning: use of deprecated associated function `zip::read::ZipFile::<'a>::sanitized_name`: by stripping `..`s from the path, the meaning of paths can change.
                You must use a sanitization strategy that's appropriate for your input
  --> src/cmd/new.rs:78:34
   |
78 | ...                   file.sanitized_name().display()
   |                            ^^^^^^^^^^^^^^

warning: 2 warnings emitted
```

**Comment by ascjones** [comment]
Fixes for this are in #79

**Comment by ascjones** [comment]
Fixed in #82
